scipy/linalg/tests/test_support_alternative_backends.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from scipy.linalg._support_alternative_backends import array_special_func_map
+from scipy.conftest import array_api_compatible
+from scipy import linalg
+from scipy._lib._array_api import xp_assert_close, is_jax, get_array_subpackage_func
+from scipy._lib.array_api_compat import numpy as np
+
+try:
+    import array_api_strict
+    HAVE_ARRAY_API_STRICT = True
+except ImportError:
+    HAVE_ARRAY_API_STRICT = False
+
+
+@pytest.mark.fail_slow(5)
+@array_api_compatible
+# @pytest.mark.skip_xp_backends('numpy', reasons=['skip while debugging'])
+# @pytest.mark.usefixtures("skip_xp_backends")
+# `reversed` is for developer convenience: test new function first = less waiting
+@pytest.mark.parametrize('f_name_n_args', reversed(array_special_func_map.items()))
+@pytest.mark.parametrize('dtype', ['float32', 'float64'])
+@pytest.mark.parametrize('shapes', [[(10, 10), (10,)]])
```

Comment:
Sure, this makes the assumption that a valid first arg is 2d and the second should be 1d, which is true for many `linalg` functions. This can be generalized, or there is no reason why a single test needs to be parameterized over all functions. It's still more compact than translating all existing tests.